sklearn/neighbors/tests/test_sample_weight.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
```

Comment:
We should already have a test file where we test these classes. I must not create a new file.

Reply:
ok, moved to `test_neighbors.py`